src/common/Input.ts

```diff
@@ -283,6 +283,11 @@ interface MouseOptions {
   clickCount?: number;
 }
 
+interface MouseWheelOptions {
```

Comment:
you'll need an `export` on this (so that when we generate the `puppeteer.d.ts` file it gets included)

Reply:
I assume we want to export `MouseOptions` too then.